targets/csharp/make.js

```diff
@@ -279,33 +288,34 @@ function getAuthParams(apiCall, isApiInstance = false) {
 }
 
 function getRequestActions(tabbing, apiCall, isApiInstance = false) {
```

Comment:
As per our conversation, I will be making another function purely for custom api behavior with `ExecuteFunction`'s custom behavior being the first in there to avoid the fact that if we go back to return statements then `apiCall.auth === "EntityToken"` would get resolved before `apiCall.name === "ExecuteFunction"` or vice versa.